build-scripts/for-appimage/prepare-debian-ubuntu.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+set -e
+
+sudo apt update
+
+sudo DEBIAN_FRONTEND=noninteractive apt-get install -y cmake g++ pkg-config rpm file \
+  libjack-dev libfftw3-dev zlib1g-dev libasound2-dev libwavpack-dev libudev-dev \
+  libwxgtk3.0-gtk3-dev docbook-xsl xsltproc gettext po4a imagemagick patchelf libgtk-3-dev librsvg2-dev
+
```

Comment:
installing `rpm`, `file` and `libjack-dev` seems is not necessary for appimage build

Reply:
Right, that should work if there's no other hidden bug. I'll test it.